fiftyone/core/collections.py

```diff
@@ -9155,6 +9156,39 @@ def values(
         Returns:
             the list of values
         """
+
+        # If we do not need to follow insertion order, we can potentially use a
+        # covered index query to get the values directly from the index and
+        # avoid a COLLSCAN.
+        if not _enforce_natural_order:
+            field = None
+            if isinstance(field_or_expr, str):
+                field = field_or_expr
+            elif isinstance(field_or_expr, list) and len(field_or_expr) == 1:
```

Comment:
The method signature promises that `field_or_expr` can be a tuple, so that needs to be supported here too.

There is an `etau.is_container()` utility that exists specifically to check for any container-like thing (list, tuple, even iterable)

Reply:
good to keep in mind in the future, but since we are only currently supporting single fields, I believe the current conditional sufficies.

```
import eta.core.utils as etau

field_or_expr = ("id")
etau.is_container(field_or_expr) # False
isinstance(field_or_expr, str) # True
```

Reply:
Technically you could do this: `ds.values(("filepath",))`